android/assets/jsons/translations/French.properties

```diff
@@ -1268,7 +1268,7 @@ Minimal faith required for\nthe next [Great Prophet]: =
  # Requires translation!
 Religions to be founded: = 
  # Requires translation!
-Religious status: = 
+Religious status: = Statut religieux
```

Comment:
sounds like possibly a suboptimal choice of words - meant is current state, alors - l'état des choses, situation, condition?